server-node-new/app.js

```diff
@@ -0,0 +1,197 @@
+const createError = require("http-errors");
+const express = require("express");
+const { join } = require("path");
+const cookieParser = require("cookie-parser");
+const logger = require("morgan");
+
+const indexRouter = require("./routes/index");
+const pingRouter = require("./routes/ping");
+
+const { json, urlencoded } = express;
+
+var app = express();
+
+// Database Setup
+
+const mongoose = require("mongoose");
+const mongoDB = "mongodb://localhost/test";
```

Comment:
you might want to store this in an environment variable so that it's easy to xhange in the future